docs/sending-commands/ssh.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Over SSH
+---
+
+The container can host an SSH console. It is enabled by setting `ENABLE_SSH` to `true`.
+The SSH server only supports password based authentication. The password is the same as the RCON password.
+
+!!! question
+    See [the RCON password](../configuration/server-properties.md/#rcon-password) section under configuration/server-properties for more information on how to set an RCON password.
+
+The SSH server runs on port `2222` inside the container.
+
+??? tip "Tip: Remember to forward the SSH port to the host"
```

Comment:
I'm not a 100% sure we should recommend forwarding the port by default like this. While it is at least in principle more secure than RCON as the pipe for the password is encrypted, it provides much the same access as RCON, and should be treated like any admin access port.